baseplate/clients/redis_cluster.py

```diff
@@ -498,4 +498,38 @@ def execute_command(self, *args: Any, **kwargs: Any) -> Any:
     # pylint: disable=arguments-differ
     def execute(self, **kwargs: Any) -> Any:
         with self.server_span.make_child(self.trace_name):
-            return super().execute(**kwargs)
+
+            success = "true"
+            start_time = perf_counter()
+            labels = {
+                f"{PROM_LABELS_PREFIX}_command": "pipeline",
+                f"{PROM_LABELS_PREFIX}_client_name": self.connection_pool.connection_kwargs.get(
+                    "client_name", ""
+                ),
+                f"{PROM_LABELS_PREFIX}_database": self.connection_pool.connection_kwargs.get(
+                    "db", ""
+                ),
+                f"{PROM_LABELS_PREFIX}_type": "standalone",
+            }
+            num_reqs = len(self.command_stack)
+            commands = [cmd.args[0] for cmd in self.command_stack]
+
+            # Whether chained or pipelined it's one concurrent transaction
+            ACTIVE_REQUESTS.labels(**labels).inc()
```

Comment:
https://github.com/reddit/baseplate.py/pull/731
you can do something like this for the active requests

Reply:
Done